app/resources/css-default.template

```diff
@@ -765,6 +790,16 @@ label.option {
   width: 100%;
 }
 
+#photo_upload {
```

Comment:
Do you have any reason to special-case these? Why not applying this rule for all input[type="upload"]?